testsuite/E11.py

```diff
@@ -1,13 +1,15 @@
 #: E111
 if x > 2:
-  print x
+  print(x)
```

Comment:
I am not sure I fully understand what you mean. The E11 errors does not seem to be related to print and/or parentheses in any way.